extensions/amp-sticky-ad/1.0/amp-sticky-ad.js

```diff
@@ -58,15 +58,20 @@ class AmpStickyAd extends AMP.BaseElement {
   /** @override */
   buildCallback() {
     this.viewport_ = this.getViewport();
-
-    toggle(this.element, true);
     this.element.classList.add('i-amphtml-sticky-ad-layout');
     const children = this.getRealChildren();
     user().assert((children.length == 1 && children[0].tagName == 'AMP-AD'),
         'amp-sticky-ad must have a single amp-ad child');
 
     this.ad_ = children[0];
     this.setAsOwner(this.ad_);
+
+    this.ad_.whenBuilt().then(() => {
```

Comment:
I have a sprint task to deprecate the code of sticky 0.1. So I think it's ok to only fix this in 1.0.